core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/package.scala

```diff
@@ -10,8 +10,10 @@ import java.time.Instant
 
 package object leonardo {
   type LabelMap = Map[String, String]
-  // this value is the default for autopause, if none is specified. An autopauseThreshold of 0 indicates no autopause
+  // this value is the default for autopause, if none is specified. An autopause Threshold of 0 indicates no autopause
   val autoPauseOffValue = 0
+  // this value is the default for auto delete, if none is specified. Threshold of 0 indicates no auto delete
```

Comment:
we should update comment to indicate that this is only how we represent "indefinite" in DB

Reply:
added one more line, not sure if that is accurate. PTAL